src/web/services/deadline-extension-helper.spec.ts

```diff
@@ -51,4 +121,64 @@ describe('DeadlineExtensionHelper', () => {
         instructorDeadlines: {},
     })).toBeFalsy();
   });
+
+  it('should filter and set deadlines before given end time correctly', () => {
+    expect(Object.keys(DeadlineExtensionHelper.getDeadlinesBeforeOrEqualToEndTime(hasOngingDeadlines, timeNow)).length)
+      .toEqual(2);
+    expect(
+      Object.keys(DeadlineExtensionHelper.getDeadlinesBeforeOrEqualToEndTime(hasNoOngoingDeadlines, timeNow)).length,
+    ).toEqual(2);
```

Comment:
Seems like an additional comma here. Intentional?

```suggestion
    expect(
      Object.keys(DeadlineExtensionHelper.getDeadlinesBeforeOrEqualToEndTime(hasNoOngoingDeadlines, timeNow)).length
    ).toEqual(2);
```

Reply:
Yup, without the comma, [eslint will complain ](https://eslint.org/docs/rules/comma-dangle)

Reply:
Seems like a false positive, let's leave it then.